test/gtest/helpers.cpp

```diff
@@ -126,6 +127,16 @@ P4CTestEnvironment::P4CTestEnvironment() {
 
 namespace Test {
 
+class ParseAnnotations : public P4::ParseAnnotations {
+ public:
+    ParseAnnotations() : P4::ParseAnnotations("FrontendTest") { }
+    void postorder(IR::Annotation* annotation) override {
+        PARSE_EXPRESSION_LIST("diagnostic")
+        PARSE("id", Constant)
```

Comment:
out of curiosity does that serve any purpose since the test code itself doesn't access that annotation directly (and any invalid annotation - not a integer literal  - would be caught in the P4Runtime code)?

Reply:
Good point. I didn't fully understand what was going on in the test. Fixed in a371db8.